server/database/models/User.js

```diff
@@ -1,9 +1,9 @@
-
 const mongoose = require('mongoose');
 
 const UserSchema = mongoose.Schema({
   username: String,
   password: { type: String, required: false },
+  userairtableid: { type: String, required: false },
```

Comment:
Can you please explain this line?

Reply:
we need the user id that the airtable generate so we can get back the user items